packages/angular/src/stories/diagrams/utils.ts

```diff
@@ -1,5 +1,5 @@
 export const getTemplate = (content: string) => `
-<div class="container theme--white">
+<div class="container theme--white" style="min-height:140px;">
```

Comment:
For certain shapes, the height of the background was half the height of the shape.